redbot/core/modlog.py

```diff
@@ -508,7 +511,11 @@ def __init__(
         self.case_str = case_str
         self.guild = guild
         if kwargs:
-            log.warning("Got unexpected keys in case %s", ",".join(kwargs.keys()))
+            log.warning(
+                "Fix this using the hidden command: `modlogset fixcasetypes`:  "
```

Comment:
We should add the prefix here.
`{prefix}modlogset fixcasetypes` or since we dont have the context ... `[p]modlogset fixcasetypes`, so it's more obvious this command is to be run in Discord and not command line.

Reply:
Not sure I agree there, the logging calls it a command.

Reply:
Cli command is a command too, there's nothing indicating it's a bot command.

Reply:
Always assume the user don't know where their head is located :smirk: